### PR TITLE
Responsive design tnw

### PR DESCRIPTION
**Pull request by brytebee**
We are tasked to clone THE NEXT WEB homepage using html5 semantic tags and media queries of CSS3.

We have tried our best to come up with the current version.

Please leave a comment where modifications are necessary.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,19 +1,993 @@
...
+      <a href="">README</a>
+      <a href="">GROWTH QUARTERS</a>
+      <a href="">SHIFT</a>
+      <a href="">NEURAL</a>
+    </nav>
+    <i class="fas fa-search"></i>
+  </header>
+  <section class="cover-section">
+    <article class="cover-story big-cover">
+      <!-- <img class="big-cover-image" src="assets\images\big-cover.jpg" alt="big-cover"> -->
```
Comment (arslanbisharat) [inline index.html]: - [ ] Leaving commented code in your code base is a bad practice

File: styles.css [hunk 2]
```diff
@@ -1,3 +1,452 @@
...
+  overflow: hidden;
+  padding-top: 20%;
+}
+
+/* Last fundiing section */
+.latest-funding-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  /* padding-left: 55px; */
```
Comment (arslanbisharat) [inline styles.css]: - [ ] Leaving commented code in your code base is a bad practice

File: index.html [hunk 3]
```diff
@@ -1,19 +1,993 @@
...
+        <p class="article-title">
+          7 easy tips to negotiate your salary as a developer
+        </p>
+        <ul class="article-info">
+          <li class="">by .cult</li>
+          <i class="far fa-clock"></i>
+          <li>50 minutes ago</li>
+          <i class="fas fa-share-alt"></i>
+          <li>10</li>
+        </ul>
```
Comment (arslanbisharat) [inline index.html]: - [ ] line 56 and 58 missing li tag

File: index.html [hunk 4]
```diff
@@ -1,19 +1,993 @@
...
+      <div class="cover-details">
+        <p class="date">Tuesday — January 19, 2021</p>
+        <p class="article-title">
+          7 easy tips to negotiate your salary as a developer
+        </p>
+        <ul class="article-info">
+          <li class="">by .cult</li>
+          <i class="far fa-clock"></i>
+          <li>50 minutes ago</li>
+          <i class="fas fa-share-alt"></i>
```
Comment (arslanbisharat) [inline index.html]: - [ ]  line 73 missing li tag

File: index.html [hunk 5]
```diff
@@ -1,19 +1,993 @@
...
+    </article>
+    <article class="cover-story small-cover2">
+      <div class="cover-details">
+        <p class="date">Tuesday — January 19, 2021</p>
+        <p class="article-title">
+          7 easy tips to negotiate your salary as a developer
+        </p>
+        <ul class="article-info">
+          <li class="">by .cult</li>
+          <i class="far fa-clock"></i>
```
Comment (arslanbisharat) [inline index.html]: - [ ]  line 86 missing li tag

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Request changes 😳

I am very happy to see this project and I can see you worked hard on this project. There are only a few issues that we need to address and I am sure that you will end up with a perfect project

_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_

This is a great opportunity to learn. Microverse is here to help you. 

**Please have a look at the suggestions to get more understanding.**

- [ ] Many lines in HTML missing li tags
- [ ] You have commented code. Please remove that. 
- [ ] Your Readme is not descriptive. Please update it

#### Page structure

##### Navigation bar

###### Large version

###### Medium version changes

- [ ] TWN logo appears at the leftmost part ([link to page](https://thenextweb.com/))

###### Small version changes

- [ ] Navigation and sub-navigation row changes to a hamburger (bars) icon at the rightmost part (next to seach icon) ([link to page](https://thenextweb.com/))

##### Top 3 articles section

###### Large version

- [ ] It contains TWN logo at the top left corner of the row ([link to page](https://thenextweb.com/))

###### Medium version changes

- [ ] TWN logo disappears ([link to page](https://thenextweb.com/))

###### Small version changes

- [ ] All images take the maximum width and are aligned in one column ([link to page](https://thenextweb.com/))

##### Latest news section

###### Medium version changes

- [ ] It aligns 3 articles per row ([link to page](https://thenextweb.com/))

###### Small version changes

- [ ] It aligns 1 articles per row ([link to page](https://thenextweb.com/))
- [ ] Each article has a cover image on the left side and title on the right side of the row ([link to page](https://thenextweb.com/))

##### Latest Funding section

###### Medium version changes

- [ ] It aligns 3 articles per row ([link to page](https://thenextweb.com/))

###### Small version changes

- [ ] It aligns 1 articles per row ([link to page](https://thenextweb.com/))

##### Categories (Apps, Gear, Tech, etc) section

###### Medium version changes

- [ ] It aligns 2 category boxes per row ([link to page](https://thenextweb.com/))

###### Small version changes

- [ ] It aligns 1 category box per row ([link to page](https://thenextweb.com/))

##### Latest deals section


###### Medium version changes

- [ ] It aligns 3 deals per row ([link to page](https://thenextweb.com/))

###### Small version changes

- [ ] It aligns 1 deal per row ([link to page](https://thenextweb.com/))

##### Footer

- [ ] **It does not change through breakpoints** ([link to page](https://thenextweb.com/))
- [ ] It contains social media icons vertically and horizontally aligned ([link to page](https://thenextweb.com/))
- [ ] It contains TNW logo with a copyright paragraph ([link to page](https://thenextweb.com/))

#### Implementation details

- [ ] CSS styles should use [media queries](https://www.w3schools.com/css/css_rwd_mediaqueries.asp) for 2 breakpoints ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/breakpoints_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/))
  - No small breakpoint as it is mobile-first and the base styles are for the mobile screen. ([Screenshot on mobile-first approach](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/mobile_first.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/))
  - medium: min-width around 768px
  - large: min-width 1024px



**Please make the changes and request a re-review for the given project**

If you get stuck somewhere, don't hesitate to reach me out or schedule s 1:1 meeting with a Full-time TSE. 

Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Review by PhillipUg — CHANGES_REQUESTED** [review]
Hey team, 👋

You have done an amazing job implementing the requirements for this project.

However, there are still some changes 👇 you need to implement to approve this request 🔍

### Changes Required ♻️ 

- [ ] Most of the suggestions by the previous TSE are still not implemented. Please read through carefully and implement them step by step
> #### Page structure
> ##### Navigation bar
> ###### Large version
> ###### Medium version changes
> * [ ]  TWN logo appears at the leftmost part ([link to page](https://thenextweb.com/))
> 
> ###### Small version changes
> * [ ]  Navigation and sub-navigation row changes to a hamburger (bars) icon at the rightmost part (next to seach icon) ([link to page](https://thenextweb.com/))
> 
> ##### Top 3 articles section
> ###### Large version
> * [ ]  It contains TWN logo at the top left corner of the row ([link to page](https://thenextweb.com/))
> 
> ###### Medium version changes
> * [ ]  TWN logo disappears ([link to page](https://thenextweb.com/))
> 
> ###### Small version changes
> * [ ]  All images take the maximum width and are aligned in one column ([link to page](https://thenextweb.com/))
> 
> ##### Latest news section
> ###### Medium version changes
> * [ ]  It aligns 3 articles per row ([link to page](https://thenextweb.com/))
> 
> ###### Small version changes
> * [ ]  It aligns 1 articles per row ([link to page](https://thenextweb.com/))
> * [ ]  Each article has a cover image on the left side and title on the right side of the row ([link to page](https://thenextweb.com/))
> 
> ##### Latest Funding section
> ###### Medium version changes
> * [ ]  It aligns 3 articles per row ([link to page](https://thenextweb.com/))
> 
> ###### Small version changes
> * [ ]  It aligns 1 articles per row ([link to page](https://thenextweb.com/))
> 
> ##### Categories (Apps, Gear, Tech, etc) section
> ###### Medium version changes
> * [ ]  It aligns 2 category boxes per row ([link to page](https://thenextweb.com/))
> 
> ###### Small version changes
> * [ ]  It aligns 1 category box per row ([link to page](https://thenextweb.com/))
> 
> ##### Latest deals section
> ###### Medium version changes
> * [ ]  It aligns 3 deals per row ([link to page](https://thenextweb.com/))
> 
> ###### Small version changes
> * [ ]  It aligns 1 deal per row ([link to page](https://thenextweb.com/))
> 
> ##### Footer
> * [ ]  **It does not change through breakpoints** ([link to page](https://thenextweb.com/))
> * [ ]  It contains social media icons vertically and horizontally aligned ([link to page](https://thenextweb.com/))
> * [ ]  It contains TNW logo with a copyright paragraph ([link to page](https://thenextweb.com/))
> 
> #### Implementation details
> * [ ]  CSS styles should use [media queries](https://www.w3schools.com/css/css_rwd_mediaqueries.asp) for 2 breakpoints ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/breakpoints_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/))
>   
>   * No small breakpoint as it is mobile-first and the base styles are for the mobile screen. ([Screenshot on mobile-first approach](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/mobile_first.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/))
>   * medium: min-width around 768px
>   * large: min-width 1024px

- [ ] The Live link in the README.md file is broken.
- [ ] You have some linter errors. Always ensure you have fixed all linter errors before requesting for a review of your project

#### Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack as **Phillip Musiime**
I would be happy to help! 👩‍💻 

Happy coding 🤓

**Review by Georjane — CHANGES_REQUESTED** [review]
## Status: Changes Required 🛑 🔧 
### Hi there Team ✋🏽 
Very impressive work with the TNW site especially with the large screen size. Keep up the great work. We have a couple of changes to do to get this project approved.

- [ ] First, we need to make sure our linters are passing before submitting for a code review.

Your project looks great for the large screen size but not for the medium and small screen size.

- Navigation Bar

- [ ] TWN logo appears at the leftmost part for the medium size. See image below
img

- [ ] The navbar for small screens is different from what you have. Navigation and sub-navigation row changes to a hamburger (bars) icon at the rightmost part (next to search icon) ([link to page](https://thenextweb.com/))
- Categories (Apps, Gear, Tech, etc) section
- [ ] For small screens, It aligns 1 category box per row.

Please fix the required changes and then submit for another code review.

_Feel free to reach out to me on Slack if you have any questions, Witah Georjane_ 🤝 
Good luck on your project and Happy Coding 💻 

**Witah Georjane
Code Reviewer at Microverse Inc
[portfolio](https://witahgeorjane.tech/) | [linkedIn](https://www.linkedin.com/in/witah-georjane/)**

**Review by ebeagusamuel — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
### Hey there :wave: 
#### There are still several changes that the previous reviewers requested that are yet to be implemented. You will need to implement all the changes asked of you by the reviewers before the project can be approved. 
- [ ] You still have some linter issues, please fix them.
- [ ] The navigation bar, top 3 article section, latest news section, latest funding section, Categories (Apps, Gear, Tech, etc) section, and latest deals section all needs to be worked on. Please refer to the changes the previous reviewer asked to be made on those sections and make the changes accordingly.
- [ ] The live link in your readme is still not working.
- [ ] Also the project is supposed to be a clone, so you need to make your clone look as close as possible to the original webpage. This means you need to use the same colors, fonts etc.
#### Do request another review when you are done making the changes.
slack(@ebeagusamuel)

**Comment by brytebee** [comment]
Dear TSE and code reviewer,

Please verify your connections because the screenshots are taking from the live Github pages and not my local server.

Thanks.

**Review by ebeagusamuel — APPROVED** [review]
# Project Approved :100: 
### Hey there :wave: 
###### Congratulations on getting this project approved. You have done the changes requested and your project now meets the basic requirements. You can now merge this branch and move to the next project.
##### P.S: Please fix the broken screenshot link in your readme and also remove the live link or update it to a working link.
slack(@ebeagusamuel)

**Comment by brytebee** [comment]
Thank you. The link issue has been fixed.

**Review by arslanbisharat — COMMENTED** [review]
### Invalid Submission

This project is already approved. Please avoid submitting review request for this project and if the status of your project still "Require Changes", Please create zendesk ticket

**Pull request merged**